.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
I think we're talking past each other. I don't care how it gets built, we can remove support for `JULIA_SYSIMG_BUILD_FLAGS` if you want (better in #16059 though, not here). As long as a no-linker .ji file gets built somehow, separately from sys.dll is fine (parallel make to save CI time), and we test that Julia can start with it, I'm satisfied.